### ISSUE TITLE
Failed to install System Informer_3.0.10801.6274.

### ISSUE THREAD

**Issue by rqueizan** [winsiderss/systeminformer#1637]
### Brief description of your issue

During the installation of System Informer, version 3.0.10801.6274, the process is interrupted and does not finish successfully.

### Steps to reproduce (optional)

Download version 3.0.10801.6274 [https://systeminformer.sourceforge.io/nightly.php](url).
Proceed to install.

### Expected behavior (optional)

The installation process should complete successfully and update the version of System Informer.

### Actual behavior (optional)

The installer quits unexpectedly.

### Environment (optional)

```shell
System Informer (Nightly) 3.0.10801.6274
Windows 11 Enterprise 22H2 Build 22621.1344
```

**Comment by rqueizan** [comment]
![SI](https://user-images.githubusercontent.com/18214796/224072472-abdbc81f-b2ea-4b2f-9203-8f17b7367120.png)

**Comment by Biswa96** [comment]
Would you like to try older installer from here https://github.com/winsiderss/si-builds/releases ?

**Comment by rqueizan** [comment]
Until version 3.0.10791.6264, all installers worked correctly. This was the most recent and currently in use. I may have missed one or the other, but it is highly unlikely that it has happened.

**Comment by KarenArzumanyan** [comment]
I also install every build. The last working build was 3.0.10791.6264.

**Comment by KarenArzumanyan** [comment]
To reproduce, you can simply install 3.0.10791.6264 and try to install 3.0.10801.6274 from above

Windows 11 Pro for WorkStations 22H2 22621.1344
Windows Feature Experience Pack 1000.22639.1000.0

**Comment by rqueizan** [comment]
Fixed in version 3.0.10822.6295.
It installs and runs correctly.

**Comment by dmex** [comment]
MSVC built the setup with invalid code and didn't fail the build 🤷‍♂️
